src/test/llmnode_test.cpp

```diff
@@ -239,19 +330,20 @@ TEST_F(LLMFlowHttpTest, unaryCompletionsJsonN) {
     ASSERT_EQ(
         handler->dispatchToProcessor(endpointCompletions, requestBody, &response, comp, responseComponents, &writer),
         ovms::StatusCode::OK);
-    rapidjson::Document d;
-    d.Parse(response.c_str());
-    ASSERT_TRUE(d["choices"].IsArray());
-    ASSERT_EQ(d["choices"].Capacity(), 8);
+    parsedResponse.Parse(response.c_str());
+    ASSERT_TRUE(parsedResponse["choices"].IsArray());
+    ASSERT_EQ(parsedResponse["choices"].Capacity(), 8);
     int i = 0;
-    for (auto& choice : d["choices"].GetArray()) {
+    for (auto& choice : parsedResponse["choices"].GetArray()) {
         ASSERT_TRUE(choice["finish_reason"].IsString());
-        ASSERT_EQ(choice["index"], i++);
+        EXPECT_STREQ(choice["finish_reason"].GetString(), "length");
```

Comment:
same as above, it might change, so I don't think we should have this assertion here